vng_api_common/migrations/0006_delete_apicredential.py

```diff
@@ -0,0 +1,106 @@
+# Generated by Django 5.1.2 on 2024-10-24 13:51
+
+import logging
+
+from django.db import IntegrityError, migrations
+from django.utils.text import slugify
+
+from zgw_consumers.constants import APITypes, AuthTypes
+
+logger = logging.getLogger(__name__)
+
+
+def get_api_type(api_root: str) -> APITypes:
+    mapping = {
+        "/autorisaties/api/": APITypes.ac,
+        "/zaken/api/": APITypes.zrc,
+        "/catalogi/api/": APITypes.ztc,
+        "/documenten/api/": APITypes.drc,
+        "/besluiten/api/": APITypes.drc,
+    }
+
+    for path, _type in mapping.items():
+        if path in api_root.lower():
+            return _type
+
+    return APITypes.orc
+
+
+def migrate_credentials_to_service(apps, _) -> None:
```

Comment:
@annashamray I'm not sure if these APICredentials were actually still used, so I'm not sure if this migration is actually needed tbh

Reply:
This migrations looks very similar to `autorisaties.0003_fill_service` migration. I think they were already migrated
I don't think APICredentials are used in Open Zaak. but there is a high change that they are still used in Open Notificaties

Reply:
oops sorry I thought it was Open Zaak repo. Please ignore my comment about duplicated migration. 

I've checked and Open Notificaties indeed uses APICredential, for example to access Autorisaties API